CHANGELOG.md

```diff
@@ -2,7 +2,20 @@
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Development]
-Nothing yet!
+### Added
```

Comment:
Thanks! I maintain one for all the open-source libs I lead. So much better than a list of commits.